src/python3/cpp_simulator.cpp

```diff
@@ -51,6 +51,9 @@ void setPatchData(PatchData& data, std::string patchID, std::string origin,
     std::memcpy(data.upper.request().ptr, upper.data(), bytes);
     data.patchID = patchID;
     data.origin  = origin;
+
+    std::memcpy(data.lower.request().ptr, lower.data(), PatchData::dimension);
```

Comment:
what's on master is correct 
```
https://github.com/PHAREHUB/PHARE/blob/master/src/python3/cpp_simulator.cpp#L49
```

Reply:
conflict lines possibly?

Reply:
yeah conflict in rebase poorly handled by me :( fixed, thx